tests/integration/data/dns-provider-test/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+name: dns-provider-test
+display-name: DNS-provider Test
+description: Test for dns-provider relation
+summary: Test for dns-provider relation
+containers:
+  httpbin:
+    resource: httpbin-image
+
+resources:
+  httpbin-image:
+    type: oci-image
+    description: OCI image for httpbin (kennethreitz/httpbin)
```

Comment:
Here again... why do we need the test charm to have an associated side-car container?  I didn't see the webservice in this container used for anything